### PR TITLE
Change to using tctl desktop ls

### PR DESCRIPTION
**Pull request by webvictim**
`tctl get windows_desktop` -> `tctl desktop ls`

### CODE REVIEW

**Comment by stevenGravy** [comment]
Given this changes a command would recommend adding a changelog in this.

File: docs/pages/choose-an-edition/migrate-to-cloud.mdx [hunk 1]
```diff
@@ -323,7 +323,7 @@ your new Teleport cluster, ensure that the setup is complete.
    List all registered Windows desktops:
    
    ```code
-   $ tctl get windows_desktop
+   $ tctl desktops ls
```
Comment (zmb3) [inline docs/pages/choose-an-edition/migrate-to-cloud.mdx:326]:
```suggestion
   $ tctl desktop ls
```

Technically the command is singular. We have an alias so `desktops` will also work, but the help messages will all say `desktop`.

**Review by zmb3 — APPROVED** [review]

**Comment by zmb3** [comment]
> Given this changes a command would recommend adding a changelog in this.

This is a docs update. We don't need a changelog entry in release notes because no functional changes are included in any release.

**Review by ptgott — APPROVED** [review]

**Comment by stevenGravy** [comment]
> > Given this changes a command would recommend adding a changelog in this.
> 
> This is a docs update. We don't need a changelog entry in release notes because no functional changes are included in any release.

my mistake in reading it. sure

**Pull request merged**